packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -7,12 +7,22 @@
       <dt-tooltip
         class="d-recipe-emoji-row__tooltip"
         content-class="d-recipe-emoji-row__tooltip-content"
+        :fallback-placements="['top', 'bottom']"
         sticky="popper"
         @shown="(shown) => emojiHovered(reaction, shown)"
       >
         <span aria-hidden="true">
-          <dt-emoji-text-wrapper size="200">
-            {{ reactionLabel(reaction) }}
+          <dt-emoji-text-wrapper size="800">
+            <p class="d-recipe-emoji-row__tooltip-emoji">
+              {{ reaction.emojiUnicodeOrShortname }}
+            </p>
+            <p class="d-recipe-emoji-row__tooltip-names">
+              {{ reaction.names }}
+              <span
+                class="d-recipe-emoji-row__tooltip-label"
+                v-text="reactionLabel(reaction)"
+              />
```

Comment:
I find the regular (non-bold) text looks good in light mode, but I find it a bit hard to read in dark mode, like it doesn't have enough contrast or weight... probably a designer decision.

light
<img width="1061" height="627" alt="Screenshot 2025-08-22 at 3 25 38 PM" src="https://github.com/user-attachments/assets/277df9ce-1239-4b3f-ac75-9ea4d934e2ee" />

dark
<img width="1061" height="627" alt="Screenshot 2025-08-22 at 3 25 50 PM" src="https://github.com/user-attachments/assets/a10948cb-32cd-416a-b058-eef15c16c322" />